dlt/destinations/impl/motherduck/configuration.py

```diff
@@ -1,24 +1,29 @@
 import dataclasses
-from typing import Any, ClassVar, Final, List
+import sys
+from typing import Any, ClassVar, Final, List, Optional
 
+from dlt.version import __version__
 from dlt.common.configuration import configspec
 from dlt.common.destination.reference import DestinationClientDwhWithStagingConfiguration
 from dlt.common.destination.exceptions import DestinationTerminalException
 from dlt.common.typing import TSecretValue
 from dlt.common.utils import digest128
-from dlt.common.configuration.exceptions import ConfigurationValueError
 
 from dlt.destinations.impl.duckdb.configuration import DuckDbBaseCredentials
 
 MOTHERDUCK_DRIVERNAME = "md"
+MOTHERDUCK_USER_AGENT = f"dlt/{__version__}({sys.platform})"
```

Comment:
this you need to clarify with @rudolfix, I'm not sure what we want here, but from what I saw I think this is according to motherduck docs, right?

Reply:
So we have agreed on this in slack.